backend/server.js

```diff
@@ -1,13 +1,16 @@
 const express = require("express");
+const mongoose = require("mongoose");
 const app = express();
 const cors = require("cors");
 require("dotenv").config();
 app.use(cors());
 app.use(express.json());
 
 const source = process.env.ATLAS_CONNECTION;
+const PORT = 4000;
```

Comment:
Is this the report to run locally?

Reply:
yeah this is just for it to run locally and know what port it is using